ledger/heavy/handler/handler.go

```diff
@@ -284,3 +290,29 @@ func (h *Handler) handleGotHotConfirmation(ctx context.Context, meta payload.Met
 		logger.Debug("got confirmation: ", confirm.String())
 	}
 }
+
+func (h *Handler) handleGetLightInitialState(ctx context.Context, meta payload.Meta) {
+	startPulse, err := h.StartPulse.PulseNumber()
+	if err != nil {
+		return
+	}
+	if meta.Pulse == startPulse {
+		var IDs []insolar.JetID
+		var drops [][]byte
+		for _, id := range h.JetTree.All(ctx, startPulse) {
+			light, _ := h.JetCoordinator.LightExecutorForJet(ctx, insolar.ID(id), startPulse)
+			if light.Equal(meta.Sender) {
+				IDs = append(IDs, id)
+				drop, _ := h.DropDB.ForPulse(ctx, id, startPulse)
```

Comment:
We only need to give drops that meta.Sender is executor for.

Reply:
Try to get drop for previous pulse and same jet. If not found - try to find for parent jet.